main.css

```diff
@@ -1,33 +1,81 @@
-.card {
-    width: 300px;
+
+body {
+    margin: 0px;
+    box-sizing: border-box;
+    overflow-x: hidden;
+}
+
+footer {
+    background-color: rgba(71, 69, 66, 0.5);
+    padding: 30px;
+    margin: 50px 0 0 0;
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+}
+
+footer span {
+    display: inline-block;
+}
+
+header {
+    width: 100%;
+    background-color: black;
+    display: flex;
+    align-items: center;
+    padding: 0 50px;
+    margin-bottom: 30px;
 }
 
-.poster {
-    width: 300px;
+header img {
+    width: 150px;
 }
 
-.title {
-    font-size: 1.4rem;
-    font-weight: bold;
-    text-align: center;
+header a {
+    color: white;
+    display: block;
+    text-decoration: none;
+    padding: 20px;
+    padding-left: 100px;
+    font-family: Arial Black;
+    letter-spacing: 5px;
+    font-size: 25px;
 }
 
-.overview {
-    height: 75px;
+.card {
+    /* width: 300px; */
+    /* height: 550px; */
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
     overflow: hidden;
+    border-radius: 10px;
+    margin: 15px;
+    border: 1px solid rgb(207, 207, 207);
 }
 
-footer {
-    background-color: rgba(71, 69, 66, 0.5);
-    position: fixed;
-    bottom: 0px;
-    padding: 30px;
-    left: 0px;
+.card img {
     width: 100%;
 }
 
-header {
-    width: 100%;
-    background-color: rgba(71, 69, 66, 0.5);
-    height: 50px;
+.card h3 {
+    font-family: 'Fredoka One', cursive;
+    margin: 0;
+    padding: 15px;
+}
+
+.card p {
+    font-family: 'Fredoka One', cursive;
+    margin: 0 0 20px 0;
+    padding: 0 25px 25px 25px;
+    height: 110px;
+    overflow: hidden;
+}
+
+main {
```

Comment:
no needed style for main it should be assigned via a class that you should be able pass via parameters in your createLayout  function

Reply:
fixed